examples/ThemeMaker.py

```diff
@@ -0,0 +1,271 @@
+self.resizable(width=False, height=False)
+
+null=None #Default is null
+
+#Main Body of the json file
+json_data={
+    "color": {
+        'window_bg_color': [null, null],
+        'frame_border': [null, null],
+        'frame_low': [null, null],
+        'frame_high': [null, null],
+        'button': [null, null],
+        'button_hover': [null, null],
+        'button_border': [null, null],
+        'checkbox_border': [null, null],
+        'checkmark': [null, null],
+        'entry': [null, null],
+        'entry_border': [null, null],
+        'entry_placeholder_text': [null, null],
+        'label': [null, null],
+        'text': [null, null],
+        'text_disabled': [null, null],
+        'text_button_disabled': [null, null],
+        'progressbar': [null, null],
+        'progressbar_progress': [null, null],
+        'progressbar_border': [null, null],
+        'slider': [null, null],
+        'slider_progress': [null, null],
+        'slider_button': [null, null],
+        'slider_button_hover': [null, null],
+        'switch': [null, null],
+        'switch_progress': [null, null],
+        'switch_button': [null, null],
+        'switch_button_hover': [null, null],
+        'optionmenu_button': [null, null],
+        'optionmenu_button_hover': [null, null],
+        'combobox_border': [null, null],
+        'combobox_button_hover': [null, null],
+        'dropdown_color': [null, null],
+        'dropdown_text': [null, null],
+        'dropdown_hover': [null, null],
+        'scrollbar_button': [null, null],
+        'scrollbar_button_hover': [null, null]
+    },
+    "text": {
+    "macOS": {
+        "font": "SF Display",
+        "size": -13
+        },
+    "Windows": {
+        "font": "Roboto",
+        "size": -13
+    },
+    "Linux": {
+        "font": "Roboto",
+        "size": -13
+        }
+    },
+    "shape": {
+        "button_corner_radius": 8,
+        "button_border_width": 0,
+        "checkbox_corner_radius": 7,
+        "checkbox_border_width": 3,
+        "radiobutton_corner_radius": 1000,
+        "radiobutton_border_width_unchecked": 3,
+        "radiobutton_border_width_checked": 6,
+        "entry_border_width": 2,
+        "frame_corner_radius": 10,
+        "frame_border_width": 0,
+        "label_corner_radius": 0,
+        "progressbar_border_width": 0,
+        "progressbar_corner_radius": 1000,
+        "slider_border_width": 6,
+        "slider_corner_radius": 8,
+        "slider_button_length": 0,
+        "slider_button_corner_radius": 1000,
+        "switch_border_width": 3,
+        "switch_corner_radius": 1000,
+        "switch_button_corner_radius": 1000,
+        "switch_button_length": 0,
+        "scrollbar_corner_radius": 1000,
+        "scrollbar_border_spacing": 4
+    }
+}
+
+
+#list of widgets
+Widgets=['Window', 'Frame', 'Button', 'CheckBox', 'Entry', 'Label', 'Text', 'ProgressBar', 'Slider',
+         'Switch', 'Menu + Dropdown', 'Scrollbar']
+
+#List containing menu content

```

Comment:
Maybe you can combine these lists into a dictionary like the following:
```python
widget_attributes = {"Winndow": ["window_bg_color"], "Frame": [..., ...], ...}
```

Reply:
Done